build/main-branch-sync/rotate-secrets.sh

```diff
@@ -72,10 +81,17 @@ COLLECTIVE_SECRET=$(oc create token ${SERVICE_ACCT_NAME} -n ${COLLECTIVE_NS} --d
 
 # Update SonarCloud tokens in GitHub repos
 echo "Setting SonarCloud token on GitHub repos..."
-for REPO in $(cat repo.txt && cat repo-extra.txt); do
+for REPO in $(cat ${SCRIPT_DIR}/repo.txt && cat ${SCRIPT_DIR}/repo-extra.txt); do
   gh secret set SONAR_TOKEN -b ${SONAR_TOKEN} --repo ${REPO}
 done
 
+# Update GitHub tokens for GitHub Actions
+echo "Setting GitHub token on GitHub repos..."
+for REPO in "stolostron/governance-policy-framework"; do
```

Comment:
why we use for loop?

Reply:
Copy/paste 🙂 ... and just in case other repos need to be added. I can just set a REPO variable if that'd be simpler